src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -36,6 +36,17 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <NETSdkInformation ResourceName="ILLink_Info" />
 
+    <!-- When linking symbols, replace PDBs in the output. -->
+    <ItemGroup Condition=" '$(_TrimmerLinkSymbols)' == 'true' ">
```

Comment:
I think we need also option to drop symbols (including embedded ones) which won't collide with this option

Reply:
I'll look into always filtering out pre-link PDBs from the output - then the linker option will be used to choose between a linked PDB or no PDB. I'm wondering if we need to continue supporting the current behavior (a way to keep a pre-link PDB as part of the output)? I'm guessing not really - but it might be useful if there are any issues with PDB linking.

Reply:
> I'm wondering if we need to continue supporting the current behavior (a way to keep a pre-link PDB as part of the output)

I don't think so as such pdbs could be invalid for linked assemblies